src/main/java/com/griefcraft/lwc/LWC.java

```diff
@@ -1824,10 +1825,10 @@ private void registerCoreModules() {
             try {
                 registerModule(new Factions());
             } catch (NoClassDefFoundError e) {
-                this.log("Failed to hook into Factions!");
+                plugin.getLogger().warning("Failed to hook into Factions!");
                 this.log("Please make sure you are using an updated version of FactionsUUID.");
```

Comment:
Perhaps make these two lines also a warning for consistency?

Reply:
No, I think that would be too much warnings, only the "failed" is a warning, how to "fix" messages should not be warning, but as the name of the level: Informations.

Reply:
The reason is because these two lines are meant to be part of the same message, so it is strange to have them under different log levels.

Reply:
I think the best option is to reduce the number of warnings to a minimum amount (1) and add informational message afterwards.
But I understand your point, I will make the changes.